examples/hosts/app-integration/external-data/src/externalData.ts

```diff
@@ -6,21 +6,30 @@
 import type { IEvent } from "@fluidframework/common-definitions";
 import { TypedEventEmitter } from "@fluidframework/common-utils";
 
+/**
+ * string-parsed representation of task data, returned from {@link parseStringData}.
+ */
+interface ParsedTaskData {
```

Comment:
For the new interfaces - we typically prefix with I for interfaces (I know this is controversial, but probably better to be consistent I think).

Reply:
Oh, do we do that for *all* interfaces? I had been under the impression we used the `I` prefix for semantic interfaces (those we expect to be implemented), but not for structural ones. Happy to update.

Reply:
I went ahead and updated it to be internally consistent here, but I am curious if we have explicit guidelines about this.

Reply:
Actually, based on our guidelines [here](https://github.com/microsoft/FluidFramework/wiki/Coding-Guidelines), I believe the intention is to only use the `I` prefix for semantic interfaces. So I think I would prefer to put these back, if that seems reasonable to you.

Reply:
AFAIK we use it in all cases (haven't seen that page before) - I don't know why it should matter how it's intended to be used.  I don't feel particularly strongly about this rule so consider this feedback optional for me.